_source/_posts/2020-06-22-deploy-dotnet-container-aws-fargate.md

```diff
@@ -3,112 +3,99 @@ disqus_thread_id: 8089923701
 
-The de facto standard is Docker for all practical purposes, but just for clarity others do exist. This standardization is where the notion of a container comes from. In shipping there use to be no standardization and this made transporting items difficult because you have to figure out how to hold each item on a boat, a train, a truck. Containers provided a standard format for transport "you can put whatever you want inside the container so long as it opens this way, can bolt down this way, and are one of these dimensions." this made the logistical process of moving real-world item much more simplified. In this same way standardizing the way a container interacts with its host OS to delegate the responsibility for the execution of tasks simplifies management of the part that is important the application.
+These are the three problem areas that containers hope to solve; Ease of management, size, and speed to start. There are others, but for simplicity, we will focus on those three as I feel (IMHO) they are the most relevant to business processes. Containers differ from a traditional virtual machine in one significant way. They do not have an actual Operating System. This may come as a surprise to you, even if you might have dabbled with containers before. Containers instead have an abstraction of an operating system that provides hooks into the host operating system using standards and conventions supplied by the container system.
 
-## Hosting Containers with AWS Fargate
+The de facto standard is Docker for all practical purposes, but just for clarity, others do exist. This standardization is where the notion of a container comes from. In shipping, there was no standardization, which made transporting items difficult because you had to figure out how to hold each item on a boat, train, or truck. Containers provided a standard format for transport "you can put whatever you want inside the container so long as it opens this way, can bolt down this way, and are one of these dimensions." this made the logistical process of moving real-world items much more simplified. In this same way, standardizing the way a container interacts with its host OS to delegate the responsibility for executing tasks simplifies management of the most crucial part, the application.
```

Comment:
```suggestion
Docker is the de facto container standard for all practical purposes, but other containers exist. This notion of containers for standardization in software development comes from shipping. In shipping, there was no standardization, which made transport difficult because you had to figure out how to hold each item on a boat, train, or truck. Containers provided a standard format for moving goods: "You can put whatever you want inside the container so long as it conforms to a standard set of dimensions and can be opened and bolted down in a uniform way." This simplified the logistical process of moving real-world items. In this same way, standardizing the way a software container interacts with its host OS to delegate responsibility for executing tasks simplifies management of the most crucial part, the application.
```
This whole section you added here is excellent. I took the liberty to rearrange/rewrite some of the more complex bits, hopefully without altering or diluting the meaning, making it a little more concise and easy to follow.